src/rviz/robot/robot_link.cpp

```diff
@@ -404,11 +405,13 @@ void RobotLink::updateAlpha()
       {
         material->setSceneBlending(Ogre::SBT_TRANSPARENT_ALPHA);
         material->setDepthWriteEnabled(false);
+        use_original_material = false;
       }
       else
       {
         material->setSceneBlending(Ogre::SBT_REPLACE);
```

Comment:
These 2 lines are responsible to make every material opaque when alpha is 1. After the commit they could be actually removed, because this material is not visible in that mode because it is replaced by the original material in Line 930.